### PR TITLE
Great Expectations Examples Fix

### PR DESCRIPTION
**Pull request by micdavis**
renamed folder name
fixed readme set up steps for examples

### CODE REVIEW

**Review by JGSweets — APPROVED** [review]

File: examples/great_expectations/README.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ These scenarios provide a practical example of how expectations can be used in r
 cd examples/great_expectations/ # cd to great expectations examples
 python3 -m venv venv # create venv
 source venv/bin/activate # activate venv
-pip install git+https://github.com/great-expectations/great_expectations.git@develop#subdirectory=contrib/capitalone_dataprofiler_expectations/ # install data profiler expectation package
+pip install -e git+https://github.com/great-expectations/great_expectations.git@develop#egg=capitalone-dataprofiler-expectations\&subdirectory=contrib/capitalone_dataprofiler_expectations/ # install data profiler expectation package
```
Comment (JGSweets) [inline examples/great_expectations/README.md:23]: remove -e

**Review by JGSweets — CHANGES_REQUESTED** [review]

File: examples/great_expectations/README.md (hunk 1 above)
Comment (taylorfturner) [reply examples/great_expectations/README.md:23]: will approve once updated

File: .gitignore [hunk 2]
```diff
@@ -49,7 +49,7 @@ wheels/
 .installed.cfg
 *.egg
 MANIFEST
-examples/great_expecations/great_expectations/
+examples/great_expectations/great_expectations/
```
Comment (taylorfturner) [inline .gitignore:52]: lol good catch!

**Review by taylorfturner — APPROVED** [review]

**Comment by taylorfturner** [comment]
Great work, @micdavis! ty

**Review by tyfarnan — APPROVED** [review]

**Pull request merged**